linkerd/protocol/http/src/e2e/scala/io/buoyant/linkerd/protocol/HttpTlsEndToEndTest.scala

```diff
@@ -69,6 +70,44 @@ class HttpTlsEndToEndTest extends FunSuite with BudgetedRetries {
 
     assert(rsp.status == Status.Ok)
   }
+
+  test("client/server works with TLSv1.2 only cipher", Retryable) {
```

Comment:
Would you mind updating this e2e test such that the request passes through a Linkerd router with the appropriate protocols configured?

Reply:
Thanks for the feedback.

I think this test boundary is around the Finagle integration rather than L5D. 
In my opinion this test should only fail when something in Finagle (newer version?) would change. 

If you think this kind of test would add value, we could look at introducing it in another part of the project, I tend to add as fewer as possible as they can fail for multiple reasons which makes debugging really hard and slow down the feedback loop.